web/src/components/navbar/NavBar.test.tsx

```diff
@@ -43,6 +55,38 @@ const setLargeScreen = (value: boolean): void => {
   (useMediaQuery as jest.Mock).mockImplementation(() => value);
 };
 
+const generateOnboardingUser = (): UserData => {
+  return generateUserData({
+    profileData: generateProfileData({
+      businessName: "",
+      industryId: undefined,
+      legalStructureId: undefined,
+    }),
+  });
+};
+
+const generateGuestUserData = (overrides?: Partial<UserData>): UserData => {
+  return generateUserData({
+    profileData: generateProfileData({
+      businessName: "",
+      industryId: "industryId",
+      legalStructureId: "legalStructureId",
```

Comment:
What's going on with these value? They're not valid industry / legal structure IDs and would cause code to break if we ever try to interpret them

Reply:
Also in the chunk of code below as well.

Reply:
Yeah, I should probably use real values.